cypress/e2e/sign_in.js

```diff
@@ -0,0 +1,91 @@
+import {sigh_in_selectors as sigh_in_selectors} from "../selectors/sigh_in_selectors";
+
+describe('UI tests for sign in page',
+    () => {
```

Comment:
Function should be in the same line with the describe

Reply:
Moved function to the same like with the describe, please review